INSTALL.md

```diff
@@ -35,18 +35,24 @@ Disabling JIT is **essential** for use with Kosmtik and other style development
 JIT can be disabled with `psql -d gis -c 'ALTER SYSTEM SET jit=off; SELECT pg_reload_conf();'` or any other means of adjusting the PostgreSQL config.
 
 ### Custom indexes
-Custom indexes are required for rendering performance and are essential on full planet databases.
+Custom indexes are required for rendering performance and are essential on full planet databases. These are generated by the `script/indexes.py` script, see `script/indexes.py --help` for various advanced options, but the command below will work to create the indexes on a new installation:
 
-```
+```sh
 psql -d gis -f indexes.sql
 ```
 
+The indexes can be created in parallel with
+
+```sh
+scripts/indexes.py -0 | xargs -0 -P0 -I{} psql -d gis -c "{}"
```

Comment:
Maybe add some comments here what the different arguments to 'indexes.py' / xargs mean? E.g., it is not immediately obvious 'P0' refers to the number of parallel threads, and must be changed to the desired number, if I understood it well from some other remark I saw you make elsewhere.

Reply:
The text above mentions `scripts/indexes.py --help`. I don't want to go into explaining xargs here - the user can just run the above command, and if they want to modify it, they'll need to understand more about what xargs does than can be reasonably explained here.